src/main/java/org/acm/auth/commands/PollCommand.java

```diff
@@ -0,0 +1,63 @@
+package org.acm.auth.commands;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import org.acm.auth.utils.Emoji;
+
+import java.awt.*;
+
+public class PollCommand extends Command{
+
+    // static object for empty perms to be used in the default constructor
+    private static final Permission[] NO_PERMS = {};
+
+    public PollCommand() {
+        super(
+                "poll",
+                "Creates a poll",
+                true,
+                false,
+                new String[] {"vote"},
+                3,
+                20, // Maximum Number of Reactions is 20, and it is kinda overkill to have more than 20 options
+                "-poll Question Option1 Option2 ... OptionN",
```

Comment:
I think the way we're using the usage string in other commands (e.g. GifCommand) is to only include the arguments and not the command name itself.
```suggestion
                "Question Option1 Option2 ... OptionN",
```